e2e/README.md

```diff
@@ -16,14 +16,15 @@ All commands/scripts in this README are executed from this folder (/e2e).
 ## Environment Variables
 
 All environment variables are optional, since they have defaults.
+For type details and defaults see [conftest.py](conftest.py)
 
-- `E2E_URL` (Default: localhost:5173)
```

Comment:
Why did you remove the documentation of the defaults? They still seem to exist in e2e/pages/constants.py?